### ISSUE TITLE
Text size limit

### ISSUE THREAD

**Issue by robot88** [PasteBar/PasteBarApp#188]
A great clipboard tool!

But I encountered some problems in the process of using it and I didn't find a solution. When the software is running in the background and monitoring the clipboard, if I copy a large content such as 10M or 50M text (of course this situation is rare), the software will be a little stuck. At this time, I right-click to view this content, and the software is stuck.

Is there any function limit that ignores the content when the copied content exceeds a certain size (such as 1M).
Looking forward to the addition of this feature.

**Comment by kisszoe2017** [comment]
I also encountered the same problem

**Comment by kurdin** [comment]
@robot88 

Yeah, this is a good idea. Thank you. I'll look at this when I have time.

**Comment by kurdin** [comment]
Closed by #198
